assets/aml-benchmark/components/batch-benchmark-inference/spec.yaml

```diff
@@ -93,6 +81,10 @@ inputs:
     description: The fallback value that can be used when request payload failed. If not provided, the fallback value will be an empty string.
     type: string
     optional: True
+  metadata_key:
```

Comment:
Is `metadata_key` needed any longer?

Reply:
This is still useful for the llama dataset or dataset that customer may want to skip hash. If this one is not enabled, then the hash will be used

Reply:
>  llama dataset 

I assume you meant llama model? 

If we don't expose this parameter to the user, things would still work for the user, right? Additionally, it reduces the number of parameters the user has to understand. I am thinking we can hide this parameter for now (we can always bring it back based on feedback). What do you think?

Reply:
I am just thinking this can be removed and the only key we will look up is `batch_metadata` in the component output. This will happen only if the customer is using the exact payload which contains `_batch_request_metadata` and we will convert this key to `batch_metadata` automatically in the final output.

Reply:
Sounds good. Thanks!

Reply:
removed